ethylene-core/build.gradle

```diff
@@ -2,4 +2,8 @@ plugins {
     id 'ethylene.library-conventions'
 }
 
-group 'com.github.steanky'
\ No newline at end of file
+group 'com.github.steanky'
+
+dependencies {
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
+}
```

Comment:
what is commons used for

Reply:
their reflection utils are absolutely essential for some of the object mapping components

Reply:
(i could probably implement some of the functions myself but it's extremely convenient not to do so here since it's error-prone)